src/modules/login/components/SocialLine/containers.styled.ts

```diff
@@ -0,0 +1,23 @@
+import styled from 'styled-components/native';
+import {indents} from "@constants/Typography";
+import {ViewProps} from "react-native";
+import React from "react";
+
+interface BaseContainerProps extends ViewProps {
+    children: React.ReactNode | React.ReactNode[];
+    indent: typeof indents;
+}
+
+export const TextContainer = styled.View.attrs<BaseContainerProps>((props: BaseContainerProps) => ({
+        indent: props.indent || indents.sm
+    })) ((props: BaseContainerProps) => ({
+    paddingBottom: props.indent
+}))
+
+export const ButtonsContainer = styled.View.attrs<BaseContainerProps>((props: BaseContainerProps) => ({
+    indent: props.indent || indents.md
+})) ((props: BaseContainerProps) => ({
+    flexDirection: 'row',
+    justifyContent: 'center',
+    paddingBottom: props.indent
+}))
```

Comment:
Again, you don't need to reuse it everywhere. `indent` is redundant from my point of view.